data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/paths/_eth_v1_node_peers_{peer_id}.json

```diff
@@ -0,0 +1,44 @@
+{
+  "get" : {
+    "tags" : [ "Node" ],
+    "operationId" : "getNodePeer",
+    "summary" : "Get node peer",
+    "description" : "Retrieves data about the given peer.",
+    "responses" : {
+      "200" : {
+        "description" : "Request successful",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "$ref" : "#/components/schemas/GetNodePeersResponse"
```

Comment:
its curious that this doesnt map to a new schema file...

Reply:
possibly because the peers endpoint we've already done uses the same thing, so it's probably ok actually, but we should list it to the issues to fix when we fix the other peers endpoint.

Reply:
Both schema definitions are named the same for `GetPeers` and `GetPeerById`. That might be it? Would we wan't to make them named different so that they map to seperate files?

Reply:
Actually both should be called `GetPeerResponse`